src/lib/validate-declarative-schema.js

```diff
@@ -122,7 +122,7 @@ const TaskSchema = joi.array().items(
       .error(taskError('"priority" should be an object with optional fields "level" and "label" or a function which returns the same')),
     actions: joi.array().items(
       joi.object({
-        type: joi.string().valid('report', 'contacts').optional(),
+        type: joi.string().valid('report', 'contact').optional(),
```

Comment:
The schema validation still fails when I don't provide a `form` property to the action. 
The form value is not used when the action type is `contact`, is it?

Reply:
You are right, added the change, and I'm going to create another PR to reflect the rule change in the [tasks documentation](https://docs.communityhealthtoolkit.org/apps/reference/tasks/).